staticd/static_main.c

```diff
@@ -42,7 +42,9 @@ char backup_config_file[256];
 
 bool mpls_enabled;
 
+
 zebra_capabilities_t _caps_p[] = {
+	ZCAP_NET_ADMIN, ZCAP_SYS_ADMIN, ZCAP_NET_RAW,
```

Comment:
Why does staticd need these permissions?

Reply:
I've changed privs/caps cleanup, and removed the capabilities.